modules/installing-ocp-agent-ibm-z-kvm.adoc

```diff
@@ -63,15 +63,15 @@ $ virt-install
     --memory=<memory> \
     --cpu host \
     --vcpus=<vcpus> \
-    --cdrom <agent_iso_image> \ <1>
+    --cdrom \<path to image>/<agent_iso_image> \ <1>
     --disk pool=default,size=<disk_pool_size> \
     --network network:default,mac=<mac_address> \
     --graphics none \
     --noautoconsole \
     --os-variant rhel9.0 \
     --wait=-1
 ----
-<1> For the `--cdrom` parameter, specify the location of the ISO image on the HTTP or HTTPS server.
+<1> For the `--cdrom` parameter, specify the location of the ISO image on the local server.
```

Comment:
```suggestion
<1> For the `--cdrom` parameter, specify the location of the ISO image on the local server, for example, `<path_to_image>/home/<image>.iso`.
```

Sorry I happened to notice this comment https://github.com/openshift/openshift-docs/pull/91632/files#r2030564236 

Should the given example be mentioned here?